teach/static/admin/js/toggle_clubs_guides_translation.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    var $ = django.jQuery.bind(django.jQuery);
+
+    function allowChoosingTranslation() {
+        $(".field-category").hide();
+        $(".field-translation_of").show();
+    }
+
+    function disallowChoosingTranslation() {
+        $(".field-category").show();
+        $(".field-translation_of").hide();
+    }
+
+    document.addEventListener("DOMContentLoaded", function() {
+        var isTranslationYes = $("#id_is_translation_0");
```

Comment:
I'd strongly advise renaming these. Make the first one `useCategory`, and the second one `translationOf`, to make it clear what these values represent.